homeassistant/components/pvpc_hourly_pricing/sensor.py

```diff
@@ -30,8 +31,9 @@
     SensorEntityDescription(
         key="PVPC",
         icon="mdi:currency-eur",
-        native_unit_of_measurement=f"{CURRENCY_EURO}/{UnitOfEnergy.KILO_WATT_HOUR}",
+        device_class=SensorDeviceClass.MONETARY,
```

Comment:
I'm not sure this is correct.
The documentation states that the unit should be a currency, not a price.
I feel that possibly [energyzero](https://github.com/home-assistant/core/blob/f9dcb2ea84b03fdd11bb532b024ca063f6324bb6/homeassistant/components/energyzero/sensor.py#L64) is wrong about that one.
```
    """Amount of money.
    Unit of measurement: ISO4217 currency code
    See https://en.wikipedia.org/wiki/ISO_4217#Active_codes for active codes
    """
```

Reply:
I'll close this one 👍